### ISSUE TITLE
Is it possible to download the file depending on HTTP response code?

### ISSUE THREAD

**Issue by orlybg** [PixelsCommander/Download-File-JS#3]
I have the case where I download a file from an API call (it gets generated on the fly) but first I need to check if there's actually data for the filters I apply and send in the API request.

Is this possible with  Download-File-JS?

Some more details on this:
http://stackoverflow.com/questions/22649087/download-file-using-jquery-from-an-api

**Comment by PixelsCommander** [comment]
No, DownloadJS is a simple piece of code, it is not possible however I have nothing against adding some new features. Just drop some considerations how API should look like.
